.github/workflows/master.yml

```diff
@@ -59,19 +58,19 @@ jobs:
         toolchain: ${{ matrix.toolchain }}
         args: --target ${{ matrix.target }} --workspace
 
-    # If musl, compile and test all excluding kafka
+    # If musl, compile and test all
     - uses: actions-rs/cargo@v1
       if: matrix.target == 'x86_64-unknown-linux-musl'
       with:
-        command: build
         toolchain: ${{ matrix.toolchain }}
-        args: --target ${{ matrix.target }} --workspace --exclude cloudevents-sdk-rdkafka
+    - run: CC=musl-gcc CXX=g++ cargo build --target ${{ matrix.target }} --all
```

Comment:
Uhm do we really need `CC` and `CXX` env vars?

Reply:
yeah, it didnt work without setting those :(